components/ApplyToHostModal.js

```diff
@@ -447,10 +447,11 @@ ApplyToHostModal.propTypes = {
   onSuccess: PropTypes.func,
   /** Use this to force the value for `collective`. If not specified, user's administrated collectives will be displayed instead */
   collective: PropTypes.object,
+  router: PropTypes.object,
 };
 
 ApplyToHostModal.defaultProps = {
   show: true,
 };
 
-export default ApplyToHostModal;
+export default withRouter(ApplyToHostModal);
```

Comment:
:ice_cream: Not a requested change, just FYI we can also use `useRouter` for functional components

Reply:
Yes, I was contemplating on if I should make that distinction between functional and non-functional components. But I opted for `withRouter` for simplicity so I have a bit of less work and we have the same pattern everywhere. But if you do see a usefulness in using `useRouter` specifically for functional components let me know and I'd be happy to make that change. 😄